sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -262,3 +262,12 @@ def get_last_enqueued_event_properties(event_data):
         }
         return event_data._last_enqueued_event_properties
     return None
+
+def parse_sas_credential(credential):
+    # type: (AzureSasCredential) -> EventHubSASTokenCredential
+    try:
+        sas = credential.signature
+        expiry = (3600 - 900) * 1000
```

Comment:
In my understanding expiry is part of the sas signature?

the sas signature has the format 'SharedAccessSignature sr={}&sig={}&se={}&skn={}' so that we could extract the expiry info from the the sigatures.

But we may need to check how other languages are doing this. (at least not a const :P)

Reply:
updated :D